### PR TITLE
Upgrade to Rails 5

### PR DESCRIPTION
**Pull request by jessieay**
Let me upgrade ya!

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -41,15 +49,16 @@ end
 
 group :test do
   gem "poltergeist"
+  gem "capybara", ">= 2.6.2"
   gem "database_cleaner"
-  gem "shoulda-matchers", require: false
+  gem "shoulda-matchers"
   gem "simplecov", require: false
-  gem "sinatra"
+  gem "sinatra", github: "sinatra/sinatra"
```
Comment (afeld) [inline Gemfile]: I'd leave comments above each of these saying why you're not on the published gem (or a link to an issue or whatever), just so people know what to look for when upgrading.

Comment (jessieay) [inline Gemfile]: good idea!

File: Gemfile [hunk 2]
```diff
@@ -2,37 +2,45 @@ source "https://rubygems.org"
...
+gem "normalize-rails", "~> 3.0.0"
 gem "omniauth-myusa"
-gem "paranoia"
+gem "paranoia", github: "rubysherpas/paranoia", branch: "core"
 gem "pg"
 gem "puma"
 gem "rack-canonical-host"
-gem "rails", "~> 4.2.0"
-gem "sass-rails"
+gem "rails", "~> 5.0.0"
```
Comment (dogweather) [inline Gemfile]: I like to see Rails first, and then the rest of the gems in alpha order. Basically, continuing the common practice established in the default Gemfile.

Comment (jessieay) [inline Gemfile]: good idea! updated at c5debb0

File: Gemfile (hunk 1 above)
Comment (jessieay) [inline Gemfile]: updated at c5debb0

Comment (afeld) [inline Gemfile]: My suggestion was to add the comments for every line where it isn't simply doing `gem "thing"`.

Comment (jessieay) [inline Gemfile]: ah yes I forgot one -- updated at 276602d

**Comment by afeld** [comment]
Just one Code Climate issue, but feel free to merge after that!

**Comment by jessieay** [comment]
@afeld thank you! waiting to merge another large-ish PR (micropurchase) will and will merge this after.

**Pull request merged**